x-pack/plugins/cases/public/components/user_action_tree/index.test.tsx

```diff
@@ -205,53 +196,34 @@ describe.skip('UserActionTree ', () => {
       </TestProviders>
     );
 
-    await waitFor(() => {
-      expect(
-        wrapper
-          .find(
-            `[data-test-subj="comment-create-action-${props.data.comments[0].id}"] [data-test-subj="user-action-markdown-form"]`
-          )
-          .exists()
-      ).toEqual(false);
-
-      wrapper
-        .find(
-          `[data-test-subj="comment-create-action-${props.data.comments[0].id}"] [data-test-subj="property-actions-ellipses"]`
-        )
-        .first()
-        .simulate('click');
-
-      wrapper.update();
-
-      wrapper
-        .find(
-          `[data-test-subj="comment-create-action-${props.data.comments[0].id}"] [data-test-subj="property-actions-pencil"]`
-        )
-        .first()
-        .simulate('click');
-
-      expect(
-        wrapper
-          .find(
-            `[data-test-subj="comment-create-action-${props.data.comments[0].id}"] [data-test-subj="user-action-markdown-form"]`
-          )
-          .exists()
-      ).toEqual(true);
+    wrapper
+      .find(
+        `[data-test-subj="comment-create-action-${props.data.comments[0].id}"] [data-test-subj="property-actions-ellipses"]`
+      )
+      .first()
+      .simulate('click');
+    wrapper
+      .find(
+        `[data-test-subj="comment-create-action-${props.data.comments[0].id}"] [data-test-subj="property-actions-pencil"]`
+      )
+      .first()
+      .simulate('click');
 
-      wrapper
-        .find(
-          `[data-test-subj="comment-create-action-${props.data.comments[0].id}"] [data-test-subj="user-action-cancel-markdown"]`
-        )
-        .first()
-        .simulate('click');
+    wrapper
+      .find(
+        `[data-test-subj="comment-create-action-${props.data.comments[0].id}"] [data-test-subj="user-action-cancel-markdown"]`
+      )
+      .first()
+      .simulate('click');
 
+    await waitFor(() => {
       expect(
         wrapper
           .find(
             `[data-test-subj="comment-create-action-${props.data.comments[0].id}"] [data-test-subj="user-action-markdown-form"]`
           )
           .exists()
-      ).toEqual(false);
```

Comment:
Might it be better to keep these more explicit. So false instead of falsy?

Reply:
you got it